Dockerfile

```diff
@@ -1,12 +1,8 @@
-FROM centos:8
-
-
-RUN sed -i 's/mirrorlist/#mirrorlist/g' /etc/yum.repos.d/CentOS-* && \
-    sed -i 's|#baseurl=http://mirror.centos.org|baseurl=http://vault.centos.org|g' /etc/yum.repos.d/CentOS-*
-
+FROM amazonlinux:2
 
 RUN yum update -y && \
-    yum module install nodejs:16/development -y && \
+    curl --silent --location https://rpm.nodesource.com/setup_16.x  | bash - && \
+    yum -y install nodejs && \
```

Comment:
This is superfluous as `nodejs` is already included in the list of packages to be installed.